ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/PackageCommitter.scala

```diff
@@ -162,6 +162,8 @@ private[kvutils] case class PackageCommitter(engine: Engine)
     "buildLogEntry" -> buildLogEntry
   )
 
+  override lazy val committerName = "packageUpload"
```

Comment:
make it "package_upload" to follow normal conventions.

Reply:
Done